IoTDeviceSimulator/web-client/src/DeviceTwin.js

```diff
@@ -49,17 +51,15 @@ export function DeviceTwin(props) {
             phenomenon: props.device.phenomenon ?? '',
             valueIsBool: props.device.valueIsBool ?? false,
             telemetrySendInterval: props.device.telemetrySendInterval ?? '',
-            behaviour: props.device.behaviour ?? '',
-            noise_magnitude: props.device.noise_magnitude ?? '',
             noiseSd: 0.45,
-            sine_period: props.device.sine_period ?? '',
-            mean: props.device.mean ?? '',
-            amplitude: props.device.amplitude ?? '',
             isRunning: props.device.isRunning ?? false,
             min: props.device.min ?? '',
             max: props.device.max ?? '',
             thingTypeName: props.device.thingTypeName ?? '',
+            currDataArray: props.device.currDataArray ?? [],
+            signalArray: props.device.signalArray ?? [`{"Behaviour":"Constant","Mean":100}`, `{"Behaviour":"Noise","Noise Magnitude":5,"Noise Standard-deviation":0.45}`],
         });
+        setTabCount(props.device.signalArray?.length ?? 2);

```

Comment:
can setTabCount be avoided here and tabcount be initialized in the declaration statement itself?

Reply:
WIP. 
Committing changes in another PR.